src/extension/actions.js

```diff
@@ -151,6 +151,20 @@ async function getAuthInfo(message, sender) {
     .map(({ owner }) => owner);
 }
 
+/**
+ * Returns the configured sites.
+ * @returns {Promise<Object[]>} The sites
+ */
+async function getSites(message, sender) {
+  const { origin } = new URL(sender.url);
+
+  if (!isTrustedOrigin(origin)) {
+    return []; // don't give out any information
+  }
+
+  return await getConfig('sync', 'projects') || [];
```

Comment:
Could drop the `await` here

Reply:
I don't think i can because of the `|| []`?